Brocfile.js

```diff
@@ -17,8 +17,8 @@ var app = new EmberAddon();
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
-app.import('bower_components/jquery-ui/jquery-ui.js');
-app.import('bower_components/jquery-ui/ui/slider.js');
-app.import('bower_components/jquery-ui/themes/base/slider.css');
+app.import('bower_components/jquery-ui/ui/minified/jquery-ui.custom.min.js');
```

Comment:
This is potentially a breaking change.  

Also, I would leave this up to the consumer and just provide documentation on how to update the Brocfile.  I can see this getting hairy if they have deps that resolve to > 1.10 they would then need to fork.  The APIs for the components haven't changed between 1.10 and 1.11 for the slider.. so I would loosen the constraint and remove these app.imports.